project/Sxr.scala

```diff
@@ -26,17 +26,22 @@ object Sxr {
     target := target.in(taskGlobal).value / "browse",
     sxr in taskGlobal := sxrTask.value
   )
-  def taskGlobal = ThisScope.copy(task = Global)
+  def taskGlobal = ThisScope.copy(task = Zero)
   def sxrTask = Def task {
```

Comment:
Style note: I feel that doing `Def task` is idiomatic and I would opt for `Def.task`. The reason why I'm nitpicking here is because our build is the best example to illustrate good practices of sbt. We use `Def.task` and `Def.setting` consistently throughout our build and ecosystem.